riff-raff/app/AppLoader.scala

```diff
@@ -8,7 +11,10 @@ class AppLoader extends ApplicationLoader {
       _.configure(context.environment)
     }
 
-    val components = new AppComponents(context)
+    val userConf = ConfigFactory.parseFile(new File(s"${scala.util.Properties.userHome}/.gu/riff-raff.conf"))
+    val contextWithNewConfig = context.copy(initialConfiguration = context.initialConfiguration ++ Configuration(userConf))
```

Comment:
Why do you need to do this now?

Reply:
Yes, because we want to use both the config from `application.conf` and the config from `riff-raff.conf`